tests/e2e/mnist_raycluster_sdk_oauth_test.py

```diff
@@ -132,12 +136,45 @@ def assert_jobsubmit_withlogin(self, cluster):
 
         client.delete_job(submission_id)
 
-        cluster.down()
-
     def assert_job_completion(self, status):
         if status == "SUCCEEDED":
             print(f"Job has completed: '{status}'")
             assert True
         else:
             print(f"Job has completed: '{status}'")
             assert False
+
+    def assert_get_cluster_and_jobsubmit(self, cluster_name, namespace):
```

Comment:
It seems to me that this code is duplicated. Can you move it to support.py where it can be shared between both tests?

Reply:
@jiripetrlik Good catch.. Refactored to support.py file. Thanks